### ISSUE TITLE
[DOC]The problem with the transition from scikit to sktime, especially for data preparation.

### ISSUE THREAD

**Issue by papaloveray** [alan-turing-institute/sktime#3097]
#### Describe the issue linked to the documentation

Hey, first of all , i'd like to thank you guys for the contribution made to time series analyze. If you want to make sktime more popular, the documentation's readability&understand-ability is so important, especially the topic like data preparation. Since most of us are  familiar with the pandas and scikit, the transition is quit important for us. 
 
I have a multi-variate classification task.
My dataframe like below:
![panel](https://user-images.githubusercontent.com/41810121/180977283-adca0d25-4e7a-4b6b-8aa9-5b34c34369eb.JPG)
Due to some reason,some symbol lost some days records, like  #1 symblol has 1000 dates rows while #2 symblol only has 500 dates rows. I want use last 11 lags to classify current label's value. For example, for the first row, I will take the OHLC from the date 2014-10-30 to  2014-10-19, to predict the date of 2014-10-30's label. 
First of all, I convert the pandas(multi-index) to sktime's mtype(pd-multiindex)
data=pd.read_csv('etf_labels.csv',encoding='utf-8',parse_dates=[1])
data.sort_values(by=['symbol','date'], ascending=[True,True],inplace=True)
data.set_index(['symbol','date'],drop=True,inplace=True)
y_train, y_test, X_train, X_test = temporal_train_test_split(y, X, test_size=30)
# fit
hc2.fit(X_train, y_train)

ValueError: y must be a np.array or a pd.Series, but found type: <class 'pandas.core.frame.DataFrame'>

if change y_train to y_train.values
hc2.fit(X_train, y_train.values)
then
ValueError: Mismatch in number of cases. Number in X = 1 nos in y = 2424

finally, i have read more than 2 days for the section of the data preparation, but still confused.

**Comment by papaloveray** [comment]
@fkiraly 
The label_11 column is my target , which has three outcomes, -1 means sell, 0 means hold, 1 means buy. 
According to the actual close price, the rolling windows length  is 11 days, we set the real label just as below:
![label](https://user-images.githubusercontent.com/41810121/181028055-ac6bf2e9-3806-4e68-a383-f4ca2e8e0edf.JPG)
Then just mentioned as earlier, i'd like to use sktime to make time series classification(sell,hold,buy).
In my opinion, it is the TSC, isn't it?
Thannks.

**Comment by fkiraly** [comment]
No, this is not time series classification, this is stream annotation in our parlance, or directional forecasting: https://github.com/alan-turing-institute/sktime/issues/763
(both experimental and not fully supported tasks - contributions appreciated!)

Why this is not TSC: there is just a single time series, and you want to attach many of those green/red dots to that time series (correpond to up/down/mid).
Do I understand correctly?
TSC is where you have, let's say, 20 time series, and to each time series you want to attach precisely one label.

**Comment by papaloveray** [comment]
@fkiraly 
To be honest, with the aim to simplify the graph, I've just showed you one series. Actually, there are more than 500 stock's time series, the big difference is the length of each time series is not the same, because of their IPO's date.
I still believe it is the TSC, precisely speaking it's interval/window-based TSC quoted by Rodríguez(2005）.
![interval](https://user-images.githubusercontent.com/41810121/181196637-aea0bbe9-4708-459f-ac43-d6eb6bbc952a.jpg)
We have spent a lot time to clarify the TSC. 
However what i really care is how to convert dataframe(pandas) pd-multiindex(sktime's mtype).  Could you give me some advice, the workable codes is preferable.
many thanks.

**Comment by fkiraly** [comment]
>I still believe it is the TSC

I think it's not:
* your multiple time series sound multivariate, not multiple instances
* there are multiple time-stamped labels per time series, not a single label per time series overall

Either way, would have to see details to be sure.

>Could you give me some advice, the workable codes is preferable.

Sure. Can you please post self-contained code that produces the data in your current format? Feel free to use dummy data.
